### ISSUE TITLE
Any idea why this Object Detection API performs badly on self-provided cat & dog samples? 

### ISSUE THREAD

**Issue by crazyqg** [tensorflow/models#2467]
Dear Expert: 
  
  The API works well with the given pre-trained models but performs extremely bad when I tried my own datasets. 
  
  I got the samples from （https://www.kaggle.com/c/dogs-vs-cats-redux-kernels-edition/data), all together 25000 pics,half dogs and half cats.   I copied them to images and rename them to cat_number.jpg & dog_number.jpg.  Also under object_detection/annotations I change the content of list.txt/trainval.txt/test.txt, with format like: 

cat_8466 1 1 1              
dog_1100 2 2 1

 I didn't touch  object_detection/annotations/trimaps as they are not used, for those files in object_detection/annotations/xmls I recreated them assigning that all pixels needs to be used. 
 

  As there are only 2 classes, so my pet_labe_map.pbtxt looks like: 
item {
  id: 1
  name: 'cat'
}

item {
  id: 2
  name: 'dog'
}
  And in the faster_rcnn_resnet101_pets.config file I didn't use "fine_tune_checkpoint:" and changed the num-classes from 37 to 2: 

  model {
  faster_rcnn {
    num_classes: 2 # 37
...
#fine_tune_checkpoint: "PATH_TO_BE_CONFIGURED/model.ckpt"

 Then I start the train with the following command:
python object_detection/train.py --logtostderr --pipeline_config_path=object_detection/faster_rcnn_resnet101_pets.config --train_dir=object_detection

  After training for 400 steps the loss decreased to almost 1: 

INFO:tensorflow:global step 1: loss = 314333248.0000 (20.141 sec/step)
INFO:tensorflow:global step 368: loss = 1.1386 (10.447 sec/step)

 After training 1200 steps I created the frozen_inference_graph.pb for test , but the result is very bad,  just wondering what mistakes have I made in the previous steps? 

  Cheers, Gang

**Comment by poxvoculi** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!
